rudder-web/src/main/scala/com/normation/rudder/web/rest/node/NodeAPIService4.scala

```diff
@@ -63,13 +64,13 @@ class NodeApiService4 (
 
   def getNodeDetails(nodeId: NodeId, detailLevel: NodeDetailLevel, state: InventoryStatus) = {
     for {
-      node          <- nodeInfoService.getNode(nodeId)
-      nodeInventory <- inventoryRepository.get(nodeId,state)
+      inventory <- inventoryRepository.get(nodeId,state)
+      node      = nodeInfoService.getNode(nodeId).getOrElse(Node(inventory))
```

Comment:
In fact it should depend on the state: 

If accepted status: getNode may fail
other status, make no request